package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "bad-words": "^3.0.4",
```

Comment:
bad-words is not needed as a dependency here, uninstall it.
`npm uninstall bad-words`

Reply:
uninstalled